components/stories/StoryFeed.tsx

```diff
@@ -1,13 +1,13 @@
 import NextLink from 'next/link'
 import { Box, Flex, Heading, SimpleGrid } from '@chakra-ui/react'
 import useTranslation from 'next-translate/useTranslation'
-import { categoryLabel, storyImage, storyCite } from './model'
+import { categories, storyImage, useLabels } from './utils'
 import ContentBox from '../common/ContentBox'
 import Label from '../common/Label'
 import { Story } from '@prisma/client'
 
 function FeedHeader() {
-  const { t } = useTranslation('common')
+  const { t } = useTranslation('story')
```

Comment:
Since the tagline is only used here, I think it belongs in story.json instead of common.json.